shell/composables/useStore.ts

```diff
@@ -0,0 +1,15 @@
+import { getCurrentInstance } from 'vue';
+
+/**
+ * useStore allows for accessing Vuex stores from within a setup function. This is a temporary measure for working with
+ * Vuex in Vue2.
+ *
+ * TODO: #9541 Remove for Vue 3 migration
```

Comment:
Since you mentioned the issue I'm working on, may I ask you what is the plan on how to remove this? 😅

Reply:
That's a good question. The expectation is that we will delete this shim and replace all usages of 

```js
 import { useStore } from '@/shell/composables/useStore';
```

with 

```js
 import { useStore } from 'vuex';
```

Will that suffice, or shall we open a new issue to deal with refactorings like this?

Reply:
Obviously not, I just needed to know what was the plan, thanks 🍻